clr-k8s-examples/create_stack.sh

```diff
@@ -63,6 +63,7 @@ function cluster_init() {
 		echo "/var/lib/etcd exists! skipping init."
 		return
 	fi
+	echo "$(kubectl kustomize ./kubeadm/metadata/)" > ./kubeadm/base/kubeadm.yaml
```

Comment:
@jascott1 a last try patching, how do this looks like? or we should move back to `sed`? as you suggest in your last comment, and let this kubeadm-kustomize gets more mature.